src/it/java/software/amazon/smithy/gradle/DisableJarTest.java

```diff
@@ -1,18 +1,3 @@
-/*
- * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Tests do not need the header, so I removed it here.

Reply:
I will add this back in though b/c these changes are adding a bunch of extra diffs and clogging up the review.